LICENSE

```diff
@@ -200,3 +199,24 @@ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
 WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
 ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
 OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+
+MIT-0 license
+-------------------------------------
+
+Copyright Amazon.com, Inc. or its affiliates.
```

Comment:
I looked at https://github.com/aws/mit-0 and it seems we need a year - do you want to add it?

Reply:
Yes, a good point. Do you know if it should be the current year (which in a sense is when it was first MIT-0 licensed) or the year range since inception of the project? That is, whether it is appropriate to write  "2024" or "2021-2024"?